artemis-core-client/src/main/java/org/apache/activemq/artemis/core/remoting/impl/netty/TransportConstants.java

```diff
@@ -61,6 +61,8 @@
 
    public static final String USE_KQUEUE_PROP_NAME = "useKQueue";
 
+   public static final String USE_IOURING_PROP_NAME = "useIoUring";
```

Comment:
Your recent comments Franz noted flagging everything with caution that its experimental...this might be a simple spot to trivially bang the point home to a user, making the option name reflect it...e.g "useIoUringExperimental"

Reply:
I disagree, that should just be clear in the docs, else we end up with mangled config.

Reply:
Having experimental bits using explicit non-final options or even APIs prior to completion to signal their status isnt mangled config IMO. Its fairly common, e.g the JDK has many instances of this.